Gateway/Request/RefundDataBuilder.php

```diff
@@ -158,7 +159,8 @@ public function build(array $buildSubject)
                     "modificationAmount" => $modificationAmount,
                     "reference" => $payment->getOrder()->getIncrementId(),
                     "originalReference" => $pspReference,
-                    "merchantAccount" => $merchantAccount
+                    "merchantAccount" => $merchantAccount,
+                    "storeId" => $payment->getOrder()->getStoreId()
```

Comment:
I know this is an easy way to do it but it is not very nice. Is there a way we can extend the transferObject and add here a new attribute storeId so we don't have to do this into the payload and unset it. @cyattilakiss remembered you did something like that before ?

Reply:
Can you check again ? @rikterbeek